x-pack/platform/plugins/shared/cases/common/types/domain/case/v1.ts

```diff
@@ -132,6 +126,7 @@ export const CaseAttributesRt = rt.intersection([
     external_service: rt.union([ExternalServiceRt, rt.null]),
     updated_at: rt.union([rt.string, rt.null]),
     updated_by: rt.union([UserRt, rt.null]),
+    incremental_id: rt.union([rt.number, rt.undefined]),
```

Comment:
@cnasikas `incremental_id` is never set to null, unlike the rest of `CaseAttributesRt`. Is this still an okay setup? `strict` and `undefined` feel icky together, but maybe that's just me ;)

Reply:
Good point. I would do the following.

```
export const CaseAttributesRt = rt.intersection([
  CaseBasicRt,
  rt.strict({
    duration: rt.union([rt.number, rt.null]),
    // rest
  }),
  rt.exact(
    rt.partial({
      incremental_id: rt.union([rt.number, rt.null]),
    })
  ),
```

The reason will also need `null` is because, due to how ES works with partial updates, you cannot "remove" a field by not setting it or setting it to `undefined`. The only option is to set it to `null`. You may need that in the next PRs, and it would be nice to be future-proof from now on. Otherwise, because the client and APIs check the schema, they will throw an error if the schema does not comply with what we have here, thus making cases inaccessible.

Reply:
@cnasikas How does that look? https://github.com/elastic/kibana/pull/222743/commits/e1c3cc75f458d8079f8978c1661f5a0ea81984c7